docs/UserGuide.md

```diff
@@ -135,10 +135,10 @@ Format: `labGrade i/INDEX l/LAB m/MARK`
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
+**Add** | `add n/NAME g/GITHUB t/TELEGRAM_HANDLE s/StudentID` <br> e.g., `add n/James Ho g/jamesH t/jamesho s/A0123456T`
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
+**Edit** | `edit INDEX [n/NAME] [g/GITHUB] [t/TELEGRAM_HANDLE] [s/StudentID]`<br> e.g.,`edit 2 n/James Lee g/jamesHo`
```

Comment:
I assume this is the end result? we going to remove the phone, email and address at the end, but for now, it's only changed here?

Reply:
Yes it's the end result. Maybe we shouldn't change it at this stage. I will revert it back to the old version.

Reply:
Sorry, just realized that our feature list already included the end result so I think following it in this section of the user guide is appropriate. Will leave it in.